source/installation/install-kafka-cluster.txt

```diff
@@ -0,0 +1,146 @@
+           <https://repo1.maven.org/maven2/io/debezium/debezium-connector-sqlserver/2.4.1.Final/debezium-connector-sqlserver-2.4.1.Final-plugin.tar.gz>`__ 
+         - `Oracle Connector plugin archive 
+           <https://repo1.maven.org/maven2/io/debezium/debezium-connector-oracle/2.4.1.Final/debezium-connector-oracle-2.4.1.Final-plugin.tar.gz>`__  
+         - `Db2 Connector plugin archive 
+           <https://repo1.maven.org/maven2/io/debezium/debezium-connector-db2/2.4.1.Final/debezium-connector-db2-2.4.1.Final-plugin.tar.gz>`__
+
+      #. Unzip the archive.
+
+      #. Copy the ``debezium-connector.jar`` file to the ``debezium-connector`` directory.
+
+         From the downloaded archive, copy the ``debezium-connector.jar`` file to the
+         ``debezium-connector-xxx`` directory at Kafka Connect’s `plugin.path 
+         <https://kafka.apache.org/documentation/#connectconfigs_plugin.path>`__ on your 
+         Kafka Connect cluster. For example: ``/kafka/connect/debezium-connector-sqlserver``.
+
+      #. Restart you Kafka Connect cluster.
+       
+         To know if your plugin is successfully registered, you must see the following 
+         log statement in the ``kafka-connect`` start up logs:
+
+         .. code-block::
+            :copyable: false
+
+            INFO || Added plugin 'io.debezium.connector.sqlserver.SqlServerConnector'
+
+   .. step:: Install Relational Migrator Sink Connect Plugin
+
+      1. Download the latest version of the ``kafka-connect.jar`` file.
+
+         To download the latest ``kafka-connect.jar`` file, see the `Migrator Installer 
+         Repository <https://migrator-installer-repository.s3.ap-southeast-2.amazonaws.com/index.html>`__.
+
+      #. Copy the ``.jar`` file to the ``kafka-connect-migrator`` directory.
+
+         The new ``kafka-connect-migrator`` directory is found at Kafka Connect’s `plugin.path 
+         <https://kafka.apache.org/documentation/#connectconfigs_plugin.path>`__. For example: 
+         ``/kafka/connect/kafka-connect-migrator``. 
+
+      #. Restart your Kafka Connect cluster. 
+
+         To know if your plugin is successfully registered, you must see the following 
+         log statement in the ``kafka-connect`` start up logs:
+
+         .. code-block::
+            :copyable: false
+
+            INFO || Added plugin 'com.mongodb.migrator.kafka.connect.MigratorSinkConnector'
+
+   .. step:: Install Relational Migrator
+
+      From the `download center <https://www.mongodb.com/try/download/relational-migrator>`__, 
+      select the appropriate Relational Migrator installer.
+
+   .. step:: Run Relational Migrator in ``kafka-integrated`` mode
+
+      To run in ``kafka-integrated`` mode, update the following configurations in your 
+      ``user.properties`` file.
+
+      Your ``user.properties`` file is located in an OS specific location. For more 
+      information, see :ref:`Relational Migrator File Locations <file-location>`.
+
+      .. list-table:: 
```

Comment:
This table looks great and has some good generic setup information which may be able to be reused on other Kafka pages. Would you consider making it an include for easier maintenance / sharability?